app/src/main/res/layout/controller_call.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?><!--
   ~ Nextcloud Talk application
   ~
-  ~ @author Mario Danic
-  ~ Copyright (C) 2017-2018 Mario Danic <mario@lovelyhq.com>
+  ~ @author Marcel Hibbe
+  ~ Copyright (C) 2021 Marcel Hibbe <dev@mhibbe.de>
```

Comment:
I'd suggest you add yourself, rather than completely eliminating me :P

Reply:
oh sorry mario, i will do that!
I guess i finally have to adjust my copyright template for Android Studio so that I don't always have to change everything manually. I suspect this happened accidentally when I tried to manually replace my name for completely new files.
no bad intention ..

- [ ] update copyright

Reply:
No bad intention was assumed ;)